src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_v1.sol

```diff
@@ -44,312 +43,394 @@ import {SafeERC20} from "@oz/token/ERC20/utils/SafeERC20.sol";
  *          to manage all the configuration for the bonding curve as well as the
  *          opening and closing of the issuance and redeeming functionalities.
  *          The contract implements the formulaWrapper functions enforced by the
- *          using the Bonding Surface formula to calculate the issuance/
+ *          using the Bonding Surface _formula to calculate the issuance/
  *          redeeming rate.
  *
  * @custom:security-contact security@inverter.network
  *                          In case of any concerns or findings, please refer to
  *                          our Security Policy at security.inverter.network or
  *                          email us directly!
  *
+ * @custom:version v1.0.0
+ *
+ * @custom:inverter-standard-version 0.1.0
+ *
  * @author  Inverter Network
  */
 contract FM_BC_BondingSurface_Redeeming_v1 is
-    IFM_BC_BondingSurface_Redeeming_v1,
-    IFundingManager_v1,
-    RedeemingBondingCurveBase_v1
+    RedeemingBondingCurveBase_v1,
+    IFM_BC_BondingSurface_Redeeming_v1
 {
     /// @inheritdoc ERC165Upgradeable
-    function supportsInterface(bytes4 interfaceId)
+    function supportsInterface(bytes4 interfaceId_)
         public
         view
         virtual
         override(RedeemingBondingCurveBase_v1)
-        returns (bool)
+        returns (bool supportsInterface_)
     {
-        return interfaceId
+        return interfaceId_
             == type(IFM_BC_BondingSurface_Redeeming_v1).interfaceId
-            || interfaceId == type(IFundingManager_v1).interfaceId
-            || super.supportsInterface(interfaceId);
+            || interfaceId_ == type(IFundingManager_v1).interfaceId
+            || super.supportsInterface(interfaceId_);
     }
 
     using SafeERC20 for IERC20;
 
-    //--------------------------------------------------------------------------
+    // ------------------------------------------------------------------------
     // Constants
 
     /// @dev Minimum collateral reserve
     uint public MIN_RESERVE;
 
-    //--------------------------------------------------------------------------
+    // ------------------------------------------------------------------------
     // Storage
 
     /// @dev The interface of the Formula used to calculate the issuance and redeeming amount.
-    IBondingSurface public formula;
+    IBondingSurface internal _formula;
     /// @dev Token that is accepted by this funding manager for deposits.
     IERC20 internal _token;
     /// @dev the amount of value that is needed to operate the protocol according to market size
     /// and conditions
-    uint public capitalRequired;
+    uint internal _capitalRequired;
     /// @dev Base price multiplier in the bonding curve formula
-    uint public basePriceMultiplier;
+    uint internal _basePriceMultiplier;
     /// @dev (basePriceMultiplier / capitalRequired)
```

Comment:
This is actually a good dev tag (showing background details of how its calculated), but needs a notice tag too.